exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_exporter.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace memory
+{
+/**
+ * A mock exporter that switches a flag once a valid recordable was received.
+ */
+class InMemorySpanExporter final : public opentelemetry::sdk::trace::SpanExporter
```

Comment:
Forgot to add this comment earlier, but implementations should be in a .cc file (except for templated classes/functions).

Reply:
Nope this was my bad. I was looking at another PR and didn't notice that this is just a class definition.